config/routes.rb

```diff
@@ -10,7 +10,7 @@
 =end
 Rails.application.routes.draw do
 
-  mount SwaggerUiEngine::Engine, :at => '/open-api'
+  mount SwaggerUiEngine::Engine, at: '/api'
```

Comment:
This seems unrelated to the Put/Patch change and has switched back and forth a few times on this file.  service_portal-api is currently using `open-api` https://github.com/ManageIQ/service_portal-api/blob/master/config/routes.rb#L13

What's the correct value here?

cc @syncrou

Reply:
I changed it to `open-api` because of the versions pr.  The base path `/api/*` wasn't being recognized correctly by the rails engine, plus I wanted to differentiate between api calls `/adi` and Swagger UI `/open-api`